eng/versioning/version_client.txt

```diff
@@ -45,6 +45,7 @@ com.azure:azure-test-watcher;1.0.0-beta.1;1.0.0-beta.1
 # Format;
 # unreleased_<groupId>:<artifactId>;dependency-version
 # note: The unreleased dependencies will not be manipulated with the automatic PR creation code.
+unreleased_com.azure:azure-core-http-jdk-httpclient;1.0.0-beta.1
```

Comment:
We don't need this unreleased tag since the current and dev version are the same, the libraries taking a dependency on this can just use the dependency tag.